controllers/cloud.redhat.com/metrics.go

```diff
@@ -0,0 +1,51 @@
+package controllers
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+var (
+	totalSuccessfulPoolReservationsCountMetrics = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "total_successful_pool_reservation_counter",
```

Comment:
I'm deep in the nitpick hole here, but I like moving units to the end of the name. [The prommy site](https://prometheus.io/docs/practices/naming/#metric-names) uses that as a general rule. In this case, `total` is the closest thing we have to units. Maybe `successful_pool_reservations_total` would be a bit more succinct.

Reply:
I totally missed the conventions on the site. That makes sense to follow.